draft-ietf-cdni-uri-signing.xml

```diff
@@ -544,6 +544,33 @@
             claim if no Client IP claim existed in the received
             signed JWT.</t>
         </section>
+        <section anchor="cdnib_claim" title="CDNI Base (cdnib) claim">
+            <t>Base (cdnir) [optional] -
+            The base path for which the token is intended. Its type is a
+            JSON string. It contains a <xref target="RFC3986">URI</xref>,
+            except that any or all of the scheme, host, or port MAY be
+            omitted. This URI MUST NOT contain User Information
+            (<xref target="RFC3986" /> Section 3.2.1), Query (Section 3.4),
+            or Fragment (Section 3.5) components.</t>
+
+            <t>The scheme is terminated by a colon (':'), which MUST be
```

Comment:
We have not yet discussed it. I'm providing specific text so that the discussion is focused.

There is definitely hidden complexity here, perhaps more than is needed. The regex, however, does not provide enough information to determine the host and path that can be used for determining what to pass when doing token renewal via cookies. If you don't need to renew the token, then the distinction isn't terribly important and regex gives you most of the tools you require already.